CMakeLists.txt

```diff
@@ -9,6 +9,9 @@ if(CMAKE_SYSTEM_NAME STREQUAL "iOS")
 endif()
 
 project(libultraship LANGUAGES C CXX)
+
+include(cmake/cvars.cmake)
```

Comment:
i'm curious as to what happens if this is included in the src cmakelists instead of the root one, it seems like something that only applies in there

Reply:
This appears to work, but would I leave `cvars.cmake` where it is, or move it inside `src`?